README.md

```diff
@@ -36,49 +36,23 @@ Run the application:
 $ oneview-redfish-toolkit
 ```
 
- At first time, it will create all the needed configuration files under user directory, and will prompt for the OneView IP(s).
+At first time, it will create all the needed configuration files under user directory, and will prompt for the OneView IP(s) to customize `redfish.conf` file with the entered IP. The following files will be created:
 
-You can update the configuration files created under the user directory, or if you want to use custom configuration files you can pass them as arguments:
+* $HOME/.config/oneview-redfish-toolkit/
+  * redfish.conf
+  * logging.conf
+  * redfish.log
```

Comment:
The log files `redfish_performance.log` and `redfish_ov_data.log` are created too.

Reply:
Done.